app/src/main/java/com/randomappsinc/studentpicker/editing/EditNameListFragment.java

```diff
@@ -159,21 +159,22 @@ void addNameWithVoice() {
 
     @Override
     public void onRenameChosen(String name) {
-        ListInfo listInfo = namesAdapter.getListInfo();
-        int currentAmount = listInfo.getInstancesOfName(name);
-        renameDialog.startRenamingProcess(name, currentAmount);
+        renameDialog.startRenamingProcess(name, amountOfEditChoiceFlow(name));
     }
 
     @Override
     public void onDeleteChosen(String name) {
-        ListInfo listInfo = namesAdapter.getListInfo();
-        int currentAmount = listInfo.getInstancesOfName(name);
-        deleteNameDialog.startDeletionProcess(name, currentAmount);
+        deleteNameDialog.startDeletionProcess(name, amountOfEditChoiceFlow(name));
     }
 
     @Override
     public void onDuplicationChosen(String name) {
-        duplicationDialog.show(name);
+        duplicationDialog.show(name, amountOfEditChoiceFlow(name));
+    }
+
+    private int amountOfEditChoiceFlow(String name) {
```

Comment:
Adding a method always comes with a cost because you want to stay underneath 65,536 methods to avoid multi-dexing, it adds latency, and it can make the code harder to read. I don't think this is worth the extra method, and the name is confusing (it has "edit" in it but is used for the "deletion" and "duplication" flow). We should continue inlining this code, and we can improve on my prior code by just merging the 2 lines together. Assigning the listinfo to a variable is kind of overkill.